build_scripts/Debian+Ubuntu/Linux_InstallGuide.md

```diff
@@ -54,17 +54,15 @@ Additional dependencies for plugins:
    libxslt-devel speex-devel speexdsp-devel
 ```
 
-#### Linux Mint
+#### Arch Linux / Manjaro / EndeavourOS
 ```bash
-   sudo apt-get install git g++ cmake qt5-qmake qtmultimedia5-dev \
-   libqt5x11extras5-dev libupnp-dev libxss-dev libssl-dev libsqlcipher-dev \
-   rapidjson-dev doxygen libbz2-dev libjson-c-dev libbotan-2-dev libasio-dev
+   sudo pacman -S base-devel libgnome-keyring cmake qt5-tools qt5-multimedia qt5-x11extras \
+   rapidjson libupnp libxslt libxss sqlcipher botan2 bzip2 json-c
```

Comment:
whenever you need rapidjson you need doxygen I think (to be verified) since the former is used for web interface and the later does the web interface calls generation.